recipe-server/package.json

```diff
@@ -109,14 +111,14 @@
     "redux-devtools-log-monitor": "1.0.11",
     "redux-logger": "3.0.6",
     "redux-mock-store": "1.1.4",
-    "sass-loader": "4.0.1",
+    "sass-loader": "6.0.6",
     "style-loader": "0.13.1",
     "stylelint": "7.6.0",
     "stylelint-config-standard": "15.0.0",
     "stylelint-order": "0.2.2",
-    "webpack": "1.13.2",
+    "uglifyjs-webpack-plugin": "^0.4.6",
```

Comment:
Took me a bit, but I got babili to compile the vendors file.

For what it's worth, babili is slower _and_ its end file size is larger - ~4.4mb after ~2 min. UglifyJS gives us ~3mb after ~50s. I understand the point of using babili across the board, but IMO it makes more sense to use the process that gives us the better compression.